src/Components/Footer.jsx

```diff
@@ -3,6 +3,11 @@ import { Grid, Header } from 'semantic-ui-react';
 import { Link } from 'react-router-dom'
 
 class Footer extends Component {
+
+refreshPage() {
+  window.location.reload()
```

Comment:
Why are you using the `location.reload`? What are you trying to achieve?

Reply:
We wanted to reload the landing 'view' when we clicked on the `Being Stockholm` logo, to trigger a re-rendering of the GET request for new posts (datapoints on map). Realized during the afternoon though that this is neither a beautiful way of doing it, nor is it in scope for this story.. Will remove this one, and instead implement re-rendering of new posts for the Map component using state updates - maybe 'leveraging' the Redux implementation that Oliver, Zane and Carla are implementing in another story - while I'm here writing this long comment.. :)